src/TBuffer.h

```diff
@@ -78,6 +78,9 @@ class TChar
         // The attributes that are currently user settable and what should be
         // consider in HTML generation:
         TestMask = 0x3f,
+        // Has been found in a search operation (currently Main Console only)
+        // and has been given a highlight to indicate that:
+        SearchFind = 0x40,
```

Comment:
How about `SearchResult`? "Search find" is a bit of a funny one

Reply:
:man_shrugging: Okay I'll try and think of something better.

Reply:
Not a fan of `SearchResult`?

Reply:
Nope! Also `Found` is shorter..  :grinning: